src/DatePicker/DateDisplay.js

```diff
@@ -113,13 +121,12 @@ class DateDisplay extends React.Component {
       DateTimeFormat,
       locale,
       selectedDate,
-      style,
       ...other,
```

Comment:
Why did we remove `style` if it's being used in `getStyles()`? May as well throw it out here instead of spreading it with `{...other}` on the root node.